To rebuild use:
  (ccl:rebuild-ccl :full t)
--- openmcl/ccl/level-1/l1-io.lisp.bb	2007-08-27 20:47:57.000000000 +0200
+++ openmcl/ccl/level-1/l1-io.lisp	2007-08-27 20:49:11.000000000 +0200
@@ -1040,6 +1040,8 @@
                         ;; or symbol consisting entirely of .'s
                         (dotimes (i len t)
                           (declare (fixnum i))
+			  (when (eql (schar name i) #\ )
+			    (return t))
                           (unless (eql (schar name i) #\.)
                             (return nil))))
                 (return-from alice
--- openmcl/ccl/level-0/l0-float.lisp.bb	2008-07-04 11:31:37.000000000 -0400
+++ openmcl/ccl/level-0/l0-float.lisp	2008-07-04 11:35:12.000000000 -0400
@@ -702,7 +702,7 @@
     (let* ((r (realpart x))
            (i (imagpart x)))
       (complex (* (cos r) (cosh i))
-               (* (sin r) (sinh i))))
+               (- (* (sin r) (sinh i)))))
     (if (typep x 'double-float)
       (%double-float-cos! x (%make-dfloat))
       #+32-bit-target
